code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -366,19 +354,19 @@
 
 /mob/living/simple_animal/drone/proc/shy_update()
 	var/list/drone_bad_areas = make_associative(drone_area_blacklist_flat) + typecacheof(drone_area_blacklist_recursive)
-	var/list/drone_good_machines = make_associative(drone_machinery_whitelist_flat) + typecacheof(drone_machinery_whitelist_recursive)
 	var/list/drone_good_items = make_associative(drone_item_whitelist_flat) + typecacheof(drone_item_whitelist_recursive)
+	var/static/list/not_shy_of = typecacheof(list(/mob/living/simple_animal/drone, /mob/living/simple_animal/bot))
```

Comment:
does this mean that drones aren't shy of pAI-controlled bots or of wizards who have shapeshifted into ED-209s?

Reply:
that's right. i think it's too much of an edge case to worry about